main/src/test/java/org/mobilitydata/gtfsvalidator/validator/StopTimeArrivalAndDepartureTimeValidatorTest.java

```diff
@@ -217,12 +228,96 @@ public void missingDepartureTimeShouldGenerateNoticeIfArrivalTimeIsProvided() {
                     GtfsTime.fromSecondsSinceMidnight(518), // arrival time
                     null,
                     "stop id",
-                    2)));
+                    2,
+                    0)));
 
     underTest.validate(noticeContainer);
     assertThat(noticeContainer.getValidationNotices())
         .containsExactly(
             new StopTimeWithOnlyArrivalOrDepartureTimeNotice(
                 1, "first trip id", 2, "arrival_time"));
   }
+
```

Comment:
I believe this is missing the false-positive check - timepoint with both times should not generate notice?